src/ProfilePage/Sidebar.jsx

```diff
@@ -335,6 +339,9 @@ return (
           </button>
         </OverlayTrigger>
       </Actions>
+      <Verifications>
+        <Widget src="${REPL_ACCOUNT}/widget/ProfilePage.ProfileBadges" props={{ accountId }} />
+      </Verifications>
```

Comment:
Seems like we don't need a `<Verifications>` wrapper here. How about to rename `ProfileBages` to `Verifications`? It's already inside of `ProfilePage` so it's understandable that this component is related to Profile, right?